docker-compose-image-redactor.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
maybe rename to `docker-compose-text` and `docker-compose-image`?

Reply:
done